integreat_cms/cms/models/poi_categories/poi_category.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..abstract_base_model import AbstractBaseModel
+
+
+class POICategory(AbstractBaseModel):
+    """
+    Data model representing a POI category.
+    """
+
```

Comment:
```suggestion

    slug = models.SlugField(
        max_length=1024,
        unique=True,
        allow_unicode=True,
        verbose_name=_("slug"),
    )
```
I guess it makes sense to also add a slug identifier in addition to the `id` field.